app/src/main/java/com/conupods/Calendar/JuilanDayConverter.java

```diff
@@ -0,0 +1,52 @@
+package com.conupods.Calendar;
+
+import android.util.Log;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class JuilanDayConverter {
+
+    private static final String TAG = "JULIAN_DAY_CONVERTER";
+    private int jd; //julian day
+    private int d;
+    private int m;
+    private int y;
+
+
+    //return a String (no need for a Date obj)
+    private void JuilanToDate() {
+        int p, q, r, s, t, u, v;
+        p = jd + 68569;
+        q = 4 * p / 146097;
+        r = p - (146097 * q + 3) / 4;
+        s = 4000 * (r + 1) / 1461001;
+        t = r - 1461 * s / 4 + 31;
+        u = 80 * t / 2447;
+        v = u / 11;
+
+        y = 100 * (q - 49) + s + v; //currently no needed, kept in case the need comes
+        m = u + 2 - 12 * v;
+        d = t - 2447 * u / 80;
+    }
+
```

Comment:
Use Meaningful variable names

Reply:
these are all Fliegel-Van Flandern's algorithm constants and manipulation variables for converting from a Julian date to a Gregorian date.  Ive stored them into final variables but they dont have a straightforward meaning outside the context of the algorithm.

Reply:
i'd leave that as a comment so that we can understand where these variables come from